client/sites-dashboard/components/sites-dashboard.tsx

```diff
@@ -78,31 +101,54 @@ export function SitesDashboard( { queryParams }: SitesDashboardProps ) {
 				</HeaderControls>
 			</PageHeader>
 			<PageBodyWrapper>
-				<ClassNames>
-					{ ( { css } ) => (
-						<SitesTableFilterTabs
-							allSites={ sites }
-							className={ css`
-								${ wideCentered }
-								position: relative;
-								top: -48px;
-							` }
-							filterOptions={ queryParams }
-						>
-							{ ( filteredSites, filterOptions ) =>
-								filteredSites.length ? (
-									<SearchableSitesTable
-										sites={ filteredSites }
-										initialSearch={ queryParams.search }
+				<SitesTableFilterTabs
```

Comment:
Are there any functional benefits to this component structure?

```js
<SitesTableFilterTabs>
  <SitesTable />
</SitesTableFilterTab>
```

With state now managed in `<SitesDashboard>`, I would've expected:

```js
<SitesTableFilterTabs />
<SitesTable />
```

My assumed answer is "there aren't any functional benefits, only a style difference, and I didn't refactor it". Is this correct?

Reply:
@danielbachhuber, I agree that now we wouldn't need to nest SitesTable, but the WordPress TabPanel we use requires us to pass some children. https://developer.wordpress.org/block-editor/reference-guides/components/tab-panel/#children